distributed/tests/test_joblib.py

```diff
@@ -140,3 +145,38 @@ def test_joblib_scatter(loop, joblib):
     assert x.count == 1
     assert y.count == 1
     assert z.count == 4
+
+
+@pytest.mark.parametrize('joblib', joblibs)
+def test_nested_backend_context_manager(loop, joblib):
+    if joblib is None or LooseVersion(joblib.__version__) <= "0.11.0":
+        pytest.skip("Joblib >= 0.11.1 required.")
+    Parallel = joblib.Parallel
+    delayed = joblib.delayed
```

Comment:
Slight preference to leave these as fully namespaced, especially given the double use of `delayed`